ios/A0RNLock/Core/A0ThemeReact.m

```diff
@@ -0,0 +1,84 @@
+//
+//  A0ThemeReact.m
+//  A0RNLock
+//
+//  Created by Julien Moutte on 27/01/2017.
+//  Copyright © 2017 Auth0. All rights reserved.
+//
+
+#import "A0ThemeReact.h"
+
+@implementation A0ThemeReact
+
+- (NSString *)getStyleKeyFromThemeKey:(NSString *)key {
+    NSString *tmp = [key substringFromIndex:7];
+    NSString *styleKey = [[[tmp substringToIndex:1] lowercaseString] stringByAppendingString:[tmp substringFromIndex:1]];
+    return styleKey;
+}
+
+- (void)themeLockColorForKey:(NSString *)key style:(NSDictionary *)style {
+    NSString *styleKey = [self getStyleKeyFromThemeKey:key];
+    NSString *value = style[styleKey];
+    if (value) {
+        unsigned rgbValue = 0;
+        NSScanner *scanner = [NSScanner scannerWithString:value];
+        [scanner setScanLocation:1]; // bypass '#' character
+        [scanner scanHexInt:&rgbValue];
+        UIColor *color = [UIColor colorWithRed:((rgbValue & 0xFF0000) >> 16)/255.0 green:((rgbValue & 0xFF00) >> 8)/255.0 blue:(rgbValue & 0xFF)/255.0 alpha:1.0];
+        [_theme registerColor:color forKey:key];
+    }
+}
+
+- (void)themeLockImageForKey:(NSString *)key style:(NSDictionary *)style {
+    NSString *styleKey = [self getStyleKeyFromThemeKey:key];
+    NSString *value = style[styleKey];
+    if (value) {
+        [_theme registerImageWithName:value bundle:NULL forKey:key];
+    }
+}
+
+- (void)themeLockWithStyle:(NSDictionary*)style {
+    _theme = [[A0Theme alloc] init];
+    
+    // Primary Button
+    [self themeLockColorForKey:@"A0ThemePrimaryButtonNormalColor" style:style];
```

Comment:
Thanks, look ok  but to make this clear that is iOS only would you wrap the style inside an `ios` attribute?

```js
style: {
  ios: {
    // iOS style customizations
  }
}
```

so that this is not mistaken with android